client/src/components/ChannelInfo/index.js

```diff
@@ -174,6 +256,50 @@ export default function ChannelInfo(props) {
           )}
         </div>
       </TabPanel>
+      <Dialog
+        open={openMembersDialog}
+        keepMounted
+        onClose={handleCloseMembersDialog}
+        aria-labelledby="alert-dialog-slide-title"
+        aria-describedby="alert-dialog-slide-description"
+        TransitionComponent={Transition}
+        maxWidth="sm"
+        fullWidth={true}
+        scroll="paper"
+      >
+        <DialogTitle>Channel Members</DialogTitle>
+        <DialogContent>
+          <Grid container spacing={2} className="channel-member-grid">
+            {channelMembers.map((user) => {
+              return (
+                <Grid
+                  key={user.username}
+                  item
+                  md={4}
+                  className="channel-member-grid-item"
+                >
+                  <img
+                    style={{ width: "30px" }}
```

Comment:
Must use classes.
The same applies to all the elements below which are directly styled.